### ISSUE TITLE
Improve 'Accessing CHT Apps': Magic Links for Android <12 only

### ISSUE THREAD

**Issue by abbyad** [medic/cht-docs#1118]
Page: /apps/concepts/access/
Version: 2022-04-21 14:42:21 -0700 -0700
---

We should specify that [Magic Links](https://docs.communityhealthtoolkit.org/apps/concepts/access/#magic-links-for-logging-in-token-login) don't currently work on Android 12+.

**Comment by Sub-codes** [comment]
I would like to work on this issue can you tell more about it

**Comment by Sub-codes** [comment]
@andrablaj can you assign this to me

**Comment by andrablaj** [comment]
I believe this ticket advise adding a note on the [Magic Links](https://docs.communityhealthtoolkit.org/building/concepts/access/#magic-links-for-logging-in-token-login) section, mentioning that it is only working for Android < 12. Do you want to give it a try @Sub-codes?

**Comment by Sub-codes** [comment]
Yes sure

**Comment by mrjones-plip** [comment]
Thanks for the follow up on an old issue @Sub-codes !  Great to get more eyes and hands on this stuff.

> We should specify that [Magic Links](https://docs.communityhealthtoolkit.org/apps/concepts/access/#magic-links-for-logging-in-token-login) don't currently work on Android 12+.

@jkuester - is this still the case?  I seem to recall that they do now work with the [updates that were made](https://github.com/medic/cht-android/issues/308#issuecomment-2364265336), but you have to follow extra steps?  I'm now forgetting 😅 

cc @paulpascal who helped here too.

**Comment by jkuester** [comment]
Yes, this issue is actually not relevant anymore since we do support deep links on Android 12+.

This is specifically noted [here](https://docs.communityhealthtoolkit.org/building/integrations/android/#sending-a-url) with instructions on how to setup the link verification [here](https://docs.communityhealthtoolkit.org/building/branding/android/#android-app-links-verification).
